defaults/main.yml

```diff
@@ -74,3 +74,6 @@ mssql_ad_kerberos_user: >-
 mssql_ad_kerberos_password: >-
   {{ ad_integration_password if ad_integration_user is defined
   else mssql_ad_sql_password }}
+
+mssql_manage_selinux: false
```

Comment:
I guess `mssql_manage_selinux` installs (or removes) the mssql selinux package, and `mssql_run_selinux_confined` says whether to run in Enforcing or Permissive if `mssql_manage_selinux` is set.  I guess it might be useful to run with selinux enabled but in Permissive mode for debugging purposes, but in general I would think if the user sets `mssql_manage_selinux: true` then they want to run with Enforcing.  If you are getting issues with selinux, or are anticipating issues, then having `mssql_run_selinux_confined` would be useful as a debugging option - perhaps then rename it as `__mssql_run_selinux_confined` so that users will know it is not for general use.

Reply:
`mssql_run_selinux_confined` decides whether to install mssql-server-selinux package, having this package ensures that SQL Server runs as a confined application, this requires selinux in Enforcing.
`mssql_manage_selinux` decides whether to change selinux mode based on what `mssql_run_selinux_confined` is set to.
Amit had the same question in the email I sent yesterday. My reply was that mssql_manage_selinux is needed for those scenarios:

> * One scenario is when users have some other management platform in addition to Ansible that they manage selinux with, we do not want Ansible to conflict with it.
> * Other scenario is to prevent the role to change selinux mode silently without user's confirmation. Consider those who set their RHEL to selinux permissive, if then role changes their selinux back to enforcing their system might throw errors from many apps and it would be hard to troubleshoot what went wrong.

WDYT?

Reply:
Also, now SQL Server 22 on RHEL 9 is in preview, GA is expected in 2024. Microsoft might implement some feedback in GA, e.g. implement checking for used ports SELinux type to match `mssql_port_t`, then mssql_manage_selinux would fix SELinux type for custom ports.